<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QWDialog</class>
 <widget class="QDialog" name="QWDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>628</width>
    <height>451</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QWDialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>GroupBox</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QCheckBox" name="chkBoxUnder">
          <property name="text">
           <string>Underline</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cheBoxBlod">
          <property name="text">
           <string>Bold</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="chkBoxItalic">
          <property name="text">
           <string>Italic</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>GroupBox</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QRadioButton" name="rBtnBlue">
          <property name="text">
           <string>Blue</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rBtnRed">
          <property name="text">
           <string>Red</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rBtnBalck">
          <property name="text">
           <string>Balck</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="contentlabel">
       <property name="text">
        <string>Content(&amp;C)</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="buddy">
        <cstring>txtEdit</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="txtEdit">
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="plainText">
        <string>Hello world,
It is my demo.</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnClose">
       <property name="text">
        <string>Exit</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnOK">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>chkBoxUnder</tabstop>
  <tabstop>cheBoxBlod</tabstop>
  <tabstop>chkBoxItalic</tabstop>
  <tabstop>txtEdit</tabstop>
  <tabstop>rBtnBlue</tabstop>
  <tabstop>rBtnBalck</tabstop>
  <tabstop>rBtnRed</tabstop>
  <tabstop>btnOK</tabstop>
  <tabstop>btnCancel</tabstop>
  <tabstop>btnClose</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>btnClose</sender>
   <signal>clicked()</signal>
   <receiver>QWDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>55</x>
     <y>430</y>
    </hint>
    <hint type="destinationlabel">
     <x>178</x>
     <y>436</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>QWDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>427</y>
    </hint>
    <hint type="destinationlabel">
     <x>424</x>
     <y>441</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnOK</sender>
   <signal>clicked()</signal>
   <receiver>QWDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>574</x>
     <y>419</y>
    </hint>
    <hint type="destinationlabel">
     <x>496</x>
     <y>421</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
